Shared/Extensions/UI Extensions/UIView+Extensions.swift

```diff
@@ -101,7 +101,7 @@ extension UIView: Selectable {
     }
 
     func set(backgroundColor: ThemeColor) {
-        self.backgroundColor = backgroundColor.color
+        self.backgroundColor = backgroundColor.color.resolvedColor(with: self.traitCollection)
```

Comment:
There's this weird thing that sometimes happens with views that are first responders, where they will pull from the wrong asset class ie light/dark. This address that. Seems like an apple hack...